jkube-kit/build/api/src/test/java/org/eclipse/jkube/kit/build/api/auth/handler/FromConfigRegistryAuthHandlerTest.java

```diff
@@ -23,22 +23,25 @@
 
 import com.google.gson.Gson;
 import com.google.gson.JsonObject;
-import mockit.Mocked;
+import org.junit.Before;
 import org.junit.Test;
-
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertThrows;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.spy;
 
 /**
  * @author roland
  * @since 23.10.18
  */
 public class FromConfigRegistryAuthHandlerTest {
 
-  @Mocked
   private KitLogger log;
-
+  @Before
+  public void setUp() throws Exception {
+    log = spy(new KitLogger.SilentLogger());
```

Comment:
```suggestion
    log = new KitLogger.SilentLogger();
```

Reply:
We don't seem to be doing any verifications. A plain `KitLogger.SilentLogger` instance should suffice